lib/referee-sinon.js

```diff
@@ -15,17 +15,23 @@ function callCount(fake) {
 }
 
 function callsToString(fake) {
-    var calls = [];
+    if (typeof fake.getCalls !== "function") {
+        return "";
+    }
 
-    for (var i = 0, l = fake.callCount; i < l; ++i) {
-        var stringifiedCall = "    " + fake.getCall(i).toString();
-        if (/\n/.test(calls[i - 1])) {
-            stringifiedCall = "\n" + stringifiedCall;
-        }
-        push.call(calls, stringifiedCall);
+    if (fake.getCalls().length === 0) {
+        return "";
     }
 
-    return calls.length > 0 ? "\n" + calls.join("\n") : "";
+    return (
+        "\n" +
+        fake
+            .getCalls()
```

Comment:
You're calling `fake.getCalls()` twice, should be a variable.

Reply:
Sure